docs/labels.md

```diff
@@ -6,88 +6,108 @@ weight: 10
 -->
 # Labels
 
-In order to make it easier to identify objects that are all part of the same
-conceptual pipeline, custom
-[labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
-set on resources used by Tekton Pipelines are propagated from more general to
-more specific resources, and a few labels are automatically added to make it
-easier to identify relationships between those resources.
+Tekton allows you to use custom [Kubernetes Labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/)
```

Comment:
NIT: I don't have strong opinion here but might be better to keep it [labels] instead of making it [Kubernetes Labels] 🤔

Reply:
I qualified it as Kubernetes labels to make it clear we're using a pre-existing entity/concept here and didn't invent our own "label" thing from scratch. If that's confusing, I can revert it.